src/styles/modal.css

```diff
@@ -0,0 +1,74 @@
+/* modal */
+
+.ui.modal {
+  background-color: var(--shade-white);
+  border-radius: 0;
+  box-shadow: none;
+}
+
+.ui.modal > .icon:first-child + * {
+  border-radius: 0;
+}
+
+/* close icon */
+
+.ui.modal > .close {
+  color: var(--shade-white);
+  font-size: 1.5rem;
+  height: 1.5rem;
+  padding: 0;
+  right: var(--spacing-2);
+  top: var(--spacing-3);
+  width: 1.5rem;
```

Comment:
Let's brighten the close button and make it a bit smaller.

```
font-size: 1rem;
width: auto;
height: auto;
right: var(--spacing-1);
top: var(--spacing-2);
opacity: 1;
```

We can also color it green on hover.

```
.ui.modal > .close:hover {
  color: var(--primary-pms-362-900);
}
```

Reply:
So, opacity changes changes on hover - it's default for all Semantic components. I don't know if we want to change this behaviour specifically for this component only or keep it consistent with the rest.
Also, I'm not sure how a green button would look like on blue background especially since it's not in the designs.

Reply:
Yeah, green on blue isn't great, but it's what we used in the component library and offhand, I don't know what else we'd use. I guess we can keep the opacity change, but let's add our `transition` to it.